nginx_module/src/http_request.rs

```diff
@@ -183,6 +190,10 @@ impl<'a> HttpRequest<'a> {
         }
     }
 
+    pub fn get_headers_out_ref(&mut self) -> &mut ngx_http_headers_out_t {
```

Comment:
I try to wrap the nginx types in safer abstractions, but it might take a lot more work to wrap ngx_http_headers_out_t in another type and expose a safe API for it ...

Reply:
i was thinking the same thing, especially that i need something for writing responses, but for now i would rather take a ref and use it.
Lets make this a todo and patch it later ?